src/site/content/en/progressive-web-apps/customize-install/index.md

```diff
@@ -0,0 +1,181 @@
+or an item in your content feed.
+
+{% Aside 'caution' %}
+You may want to wait before showing the prompt to the user so you don't
+distract them from what they're doing. For example, if the user is in a
+check-out flow, or creating their account, let them complete that before
+interrupting them with the prompt.
+{% endAside %}
+
+### In-app installation flow
+
+To provide an in-app installation flow, you'll need to provide a button,
+or other user interface element within your app that the user can click,
+indicating users can install your Progressive Web app. When the element is
+clicked, call `prompt()` on the saved `beforeinstallprompt` event (stored
+in the `deferredPrompt` variable). It shows the user a modal install dialog,
+asking them to confirm they want to install your PWA.
+
+```js
+buttonInstall.addEventListener('click', (e) => {
+  // Hide the install promotion
+  hideInstallPromotion();
+  // Show the install prompt
+  deferredPrompt.prompt();
+  // Wait for the user to respond to the prompt
+  deferredPrompt.userChoice.then((choiceResult) =>{
+    if (choiceResult.outcome === 'accepted') {
+      console.log('User accepted the install prompt');
+    } else {
+      console.log('User dismissed the install prompt');
+    }
+  })
+});
+```
+
+The `userChoice` property is a promise that resolves with the user's choice.
+You can only call `prompt()` on the deferred event once. If the user
+dismisses it, you'll need to wait until the `beforeinstallprompt` event
+is fired again, typically immediately after the `userChoice` property
+has resolved.
+
+## Detect when the PWA was successfully installed
+
+You can use the `userChoice` property to determine if the user installed
+your app from within your user interface. But, if the user installs your
+PWA from the address bar, or other browser component, `userChoice` won't
+help. Instead, you should listen for the `appinstalled` event. It is fired
+whenever your PWA is installed, no matter what mechanism is used to install
+your PWA.
+
+```js
+window.addEventListener('appinstalled', (evt) => {
+  console.log('a2hs installed');
+});
+```
+
+## Detect how the PWA was launched
+
+The CSS `display-mode` media query indicates how the PWA was launched,
+either in a browser tab, or as an installed PWA. This makes it possible to
+apply different styles depending on how the app was launched. For example,
+always hide the install button and provide a back button when launched as an
+installed PWA.
+
+### Update UI based on how the PWA was launched
+
+To apply a different background color for a PWA when launched as an installed
+PWA, use conditional CSS:
+
+```css
+@media all and (display-mode: standalone) {
+  body {
+    background-color: yellow;
+  }
+}
+```
+
+### Track how the PWA was launched
+To track how users launch your PWA, use `matchMedia()` to test the
+`display-mode` media query. Unfortunately, Safari on iOS don't support
+this yet, so you must check `navigator.standalone`.
```

Comment:
Is there a link you can provide that explains why `navigor.standalone` works for this?

Reply:
Apple's APIs aren't super easily linked, and the MDN docs for this don't really exist. I couldn't find a good link to use.

Reply:
What about adding a sentence or two to explain it?